Gosu/Window.hpp

```diff
@@ -50,11 +50,21 @@ namespace Gosu
         //! function.
         Window(unsigned width, unsigned height, bool fullscreen = false,
             double update_interval = 16.666666);
+        Window(unsigned width, unsigned height, bool fullscreen = false, bool resizable = false,
```

Comment:
Not sure about this one - I personally would just add it behind the update_interval but Overloading in C++ is fine i guess. In Ruby on the other hand I think this would be a good opportunity to introduce an options-Hash. So I can imagine something like:
```ruby
class Gosu::Window
  def initialize(width, height, fullscreen: false, update_interval: 16.666666, resizable: false)
end
```

What do you think @jlnr (btw was there a reason you did define the options-Hash logic in ruby.i instead of patches?)